app/views/api/services/forms/_integration_settings_apiap.html.slim

```diff
@@ -0,0 +1,20 @@
+#settings
+  = form.inputs "Deployment" do
+
+    - if @service.deployment_option == nil
```

Comment:
Probably Rubocop complains about this and requests `- if @service.deployment_option.nil?`
However, I'd suggest instead of these 2 lines, having: `@service.deployment_option ||= 'hosted'`

Reply:
ok, I could change it. Bare in mind this is a copy/pasta template (with minor mods) of https://github.com/3scale/porta/blob/master/app/views/api/services/forms/_integration_settings.html.slim
you might find tons of things there hehe

Reply:
oki, no problem :smile: